### DISCUSSION TITLE
Change naming of Background

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#492] [Ideas]
![image](https://user-images.githubusercontent.com/64191883/145990728-cc295472-c204-496d-ad74-5f8725a1adfa.png)

This is current layout but the background is actually the small version of the banner. Perhaps Preview banner of Card Banner is more fitting? Background tends to mean the background of a page rather than the thumbnail-ish version of a banner.

**Comment by techsmyth** [comment]
Fair point. I suggest we take this comment directly up as part of the epic related to image management, I will add a link to this thread there...

**Reply by techsmyth** [reply]
https://github.com/alkem-io/alkemio/issues/486

**Comment by techsmyth** [comment]
@Roseghost yes indeed we will be renaming background to bannerNarrow - pretty close to the top of the backlog so not far off...

**Reply by techsmyth** [reply]
@Roseghost this is planned for the current sprint, including full meta data support around every visual that is used (sizes, ratio etc)
